src/app/index.config.coffee

```diff
@@ -36,6 +36,22 @@ angular.module 'mnoEnterpriseAngular'
           $q.reject rejection
       }
 
+  .factory('PasswordExpiredInterceptor', ($log, $q,$location,$window) ->
+    return {
+      'responseError': (response) ->
+        if response.status == 403 && response.data.error && response.data.error == "Your password is expired. Please renew your password."
```

Comment:
Is it i18nable ?

Reply:
We actually match the response from the api which is not I18ned. The message we display to the user is I18ned